geode-wan/src/test/java/org/apache/geode/internal/cache/wan/GatewayReceiverImplTest.java

```diff
@@ -62,7 +62,8 @@ public void setUp() {
   @Test
   public void getHostOnUnstartedGatewayShouldReturnLocalHostName() throws UnknownHostException {
     GatewayReceiver gateway =
-        new GatewayReceiverImpl(cache, 2000, 2001, 5, 100, null, null, null, true);
+        new GatewayReceiverImpl(cache, 2000, 2001, 5, 100, null, null, null, true, a -> true,
```

Comment:
Can you change the "a->" to a better name?

Reply:
`a -> true` has been renamed to `port -> true`

`a -> 2000` has been renamed to `portRange -> 2000`

Reply:
Since the test doesn't need or use the input arguments, I changed it a 2nd time to remove them altogether.